code-of-conduct-transparency-reports/2019-07-02-coc-transparency-report.md

```diff
@@ -0,0 +1,29 @@
+# The Carpentries Code of Conduct Transparency Report
+## 2019 July 02
+
+## Overview
+
+### Reports
+
+In the last eight months, the Code of Conduct committee has not received an incident report, although the [Incident Reporting Form](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl_6rkVtBIVWbb9YNOHQP_XaANDClmVNu0zs-w/viewform) was submitted once with no information included.
+
+### Potential Code of Conduct Breaches  
+There have been no **reported** Code of Conduct breaches in the last eight months. However, discussions in the community and within the Code of Conduct committee have led the Executive Council to approve a task force that will develop recommendations for incidents that fall outside the scope of the Code of Conduct committee. 
```

Comment:
Can we change **reported** to `officially reported` (no bolding). I'd prefer we use words rather than font formatting to make this point - clearer, and there is no guessing about what we mean. 

Also recommend changing `scope of the Code of Conduct committee` to `scope of the existing Code of Conduct`. It is the document that define the scope, not the committee.